spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -0,0 +1,669 @@
+
+NEAR_ZERO_THRESHOLD = 1e-6
+
+
+# PARSER
+# ==========================================================================================
+def get_parser():
+    parser = SCTArgumentParser(
+        description='Compute normalized morphometric measures to assess spinal cord compression.\n'
+                    '\n'
+                    'Metrics are normalized using the non-compressed levels above and below the compression site using '
+                    'the following equation:\n'
+                    '\n'
+                    '    ratio = (1 - mi/((ma+mb)/2))\n'
+                    '\n'
+                    'Where mi: metric at the compression level, ma: metric above the compression level, mb: metric '
+                    'below the compression level.\n'
+                    '\n'
+                    'Additionally, if the "-normalize" flag is used, metrics are normalized using a database built '
+                    'from healthy control subjects. This database uses the PAM50 template as an anatomical reference '
+                    'system.\n'
+                    '\n'
+                    'Reference: Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic '
+                    'spinal cord injury: MR imaging findings correlated with neurologic outcome--prospective study '
+                    'with 100 consecutive patients. Radiology 2007;243(3):820-827.'
+    )
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help='Spinal cord segmentation mask to compute morphometrics. Example: sub-001_T2w_seg.nii.gz'
+             '\nNote: If no normalization is wanted (i.e., if the "-normalize" flag is not specified),'
+             ' metric ratio will take the average along the segmentation centerline.'
+    )
+    mandatoryArguments.add_argument(
+        '-vertfile',
+        metavar=Metavar.file,
+        required=True,
+        help='Vertebral labeling file. Example: sub-001_T2w_seg_labeled.nii.gz'
+             'Note: The input and the vertebral labelling file must be in the same voxel coordinate system'
+             'and must match the dimensions between each other.'
+    )
+    mandatoryArguments.add_argument(
+        '-l',
+        metavar=Metavar.file,
+        required=True,
+        help='NIfTI file that includes labels at the compression sites. '
+             'Each compression site is denoted by a single voxel of value `1`.'
+             ' Example: sub-001_T2w_compression_labels.nii.gz '
+             'Note: The input and the compression label file must be in the same voxel coordinate system '
+             'and must match the dimensions between each other.'
+    )
+    mandatoryArguments.add_argument(
+        '-metric',
+        required=False,
+        help='Metric to normalize. '
+             'Choices: area, diameter_AP, diameter_RL, eccentricity, solidity\n',
```

Comment:
hmm on my side it only displays the default: 
![image](https://user-images.githubusercontent.com/71230552/228364170-581ea213-74f1-46da-8b98-06d6c1a0d256.png)
It should be there 2 times (since I wrote it) if it showed. Maybe because of the default.

Reply:
Oh! Can you try removing the `metavar.Str` part? I think the presence of that line causes `<str>` to appear instead of the choices.

Reply:
it works: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003/commits/e029d754f54df2a6f352d9538d63e33b0221b856